### PR TITLE
Feat/integrate youtube channel

### PR DESCRIPTION
**Pull request by annlaumets**
## What changes

Integrate Youtube channel into the main page under "Videos" tab in the news feed

## Anything to note for reviewers?

- You need to add a Google API key and the playlist ID of the channel. You can find the ID of the Youtube channel (e.g. "UC_x5XG1OV2P6uZZ5FSM9Ttw") and then just change the second letter to U, so it reads "UU_x5XG1OV2P6uZZ5FSM9Ttw" instead.

## Screenshots (if applicable)

-

## Issues

-

## Resolves

- Resolves [https://app.dework.xyz/new-order/technicalengineerin?taskId=2ace4d05-b22a-47ff-b605-e948b1994db1](https://app.dework.xyz/new-order/technicalengineerin?taskId=2ace4d05-b22a-47ff-b605-e948b1994db1
)

### CODE REVIEW

File: src/components/NewsFeed.tsx [hunk 1]
```diff
@@ -101,8 +103,40 @@ const NewsFeed = () => {
...
+        return data.map((article: YoutubeArticleProps) => {
+          return {
+            id: article.id,
+            title: article.snippet.title,
+            dateTime: article.snippet.publishedAt,
+            webLink: `https://www.youtube.com/watch?v=${article.snippet.resourceId.videoId}`,
+            mediaLink: article.snippet.thumbnails.medium.url,
+            description: removeMd(article.snippet.description)
+              .replace('\n', ' ')
+              .substring(0, 200),
```
Comment (niikkiin) [inline src/components/NewsFeed.tsx:124]: can you remove the `.substring(0, 200)` here? We are already limiting the number of lines up to 3 using chakra ui's prop. thanks!

**Review by niikkiin — APPROVED** [review]
thanks for submitting a PR @annlaumets. Great work! I reviewed your code and it looks good to me. We'll just wait for you to finish my comment on the substring part and then we're good to go. cc @arjanvillon 
image

**Comment by annlaumets** [comment]
I removed the substring part from the description and pushed the changes.

**Comment by niikkiin** [comment]
This looks good to me, I will merge it now. cc @arjanvillon

**Pull request merged**